clang/test/CodeGenCoroutines/coro-alloc.cpp

```diff
@@ -1,6 +1,9 @@
 // RUN: %clang_cc1 -triple x86_64-unknown-linux-gnu -std=c++20 -O2 \
 // RUN:    -Wno-coroutine-missing-unhandled-exception -emit-llvm %s -o - -disable-llvm-passes \
 // RUN:   | FileCheck %s
+// RUN: %clang_cc1 -triple x86_64-unknown-linux-gnu -std=c++20 -O2 \
+// RUN:    -Wno-coroutine-missing-unhandled-exception -emit-llvm %s -o - -disable-llvm-passes -fcxx-type-aware-allocators \
+// RUN:   | FileCheck %s
 
```

Comment:
If I read correctly, now we can have seperate allocators for different promise types, is it? If yes, can we add some test here?

Reply:
I can do so if you tell me how (the changes in these tests were to ensure the tests pass with the feature enabled - e.g. ensuring lookup didn't fail in any overtly weird ways)